utils/build/docker/java/iast-common/src/main/resources/schema.sql

```diff
@@ -1,8 +1,15 @@
+-- Used by IAST
 CREATE TABLE USER
 (
     ID INTEGER IDENTITY PRIMARY KEY,
     USERNAME VARCHAR(32) NOT NULL,
     PASSWORD VARCHAR(32) NOT NULL
 );
 
-
+-- Used by RASP
+CREATE TABLE USERS
+(
+    ID INTEGER IDENTITY PRIMARY KEY,
+    USERNAME VARCHAR(32) NOT NULL,
+    PASSWORD VARCHAR(32) NOT NULL
+);
```

Comment:
I'm out of context, but this seems odd, apart from the fact that it can lead to misleading table names that are so similar.